packages/application-components/src/components/modal-pages/internals/modal-page.tsx

```diff
@@ -59,6 +59,7 @@ type Props = {
   baseZIndex: number;
   getParentSelector: typeof getDefaultParentSelector;
   shouldDelayOnClose: boolean;
+  afterOpenTransform?: string;
```

Comment:
I would suggest to use a more generic prop, something like `afterOpenStyles`.

So you can pass either a class name or a style object. Then in the `afterOpen` prop you do:

```js
afterOpen: typeof props.afterOpenStyles === 'string'
  ? props.afterOpenStyles
  : makeClassName(props.afterOpenStyles ?? getAfterOpenContainerAnimation())
```

The point here is that if we make the "after open" styles customizable, we should make the style itself customizable and not only a single css value.